controller/volume_controller.go

```diff
@@ -361,11 +361,6 @@ func (vc *VolumeController) syncVolume(key string) (err error) {
 		}
 
 		// now snapshots, replicas, and engines have been marked for deletion
-		if snapshots, err := vc.ds.ListVolumeSnapshotsRO(volume.Name); err != nil {
```

Comment:
After enqueueing the volume change at deletion, the snapshots should get removed. Can you explain why we are removing this dependency for volume deletion?

Reply:
Surely, there are 2 points:
1. Volume CR deletion doesn't need to wait for snapshot CR deletion
2. The DeleteFunc event handler func only gets triggered when the volume object is removed from the cache, not when the volume CR has the deletion timestamp. We might be wondering why not use the UpdateFunc instead. The reason is because UpdateFunc get triggered very frequently and waste resources watching for volume CR changes

Reply:
Thank you for the explanation. It's clear now.